docs/development-tips.md

```diff
@@ -30,9 +30,10 @@ The non-v1 apps are the following:
  - know-before-you-owe (ghe/CFGOV/know-before-you-owe) - for the Consumer Tools > Know before you owe section.
 
 After installing these projects as sibling directories to the `cfgov-refresh` repository,
-build the third-party projects per their directions,
-stop the web server and return to `cfgov-refresh`
-and run `cfgov/manage.py sheer_index -r` to load the projects' data into ElasticSearch.
```

Comment:
I think this is good enough — this whole section bothers me (not just the continued existence of the sheer apps but also the way it talks about the satellite Django apps), but we'll rewrite it in a better way later.